optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/DotNames.java

```diff
@@ -64,6 +65,7 @@
     static final DotName PLANNING_ID = DotName.createSimple(PlanningId.class.getName());
 
     static final DotName PLANNING_VARIABLE = DotName.createSimple(PlanningVariable.class.getName());
+    static final DotName PLANNING_LIST_VARIABLE = DotName.createSimple(PlanningListVariable.class.getName());
```

Comment:
Missing `IndexShadowVariable` entry (which should also be in `PLANNING_ENTITY_FIELD_ANNOTATIONS` + `GIZMO_MEMBER_ACCESSOR_ANNOTATIONS`).